src/runtime_src/core/pcie/driver/linux/xocl/devices.h

```diff
@@ -3394,6 +3413,7 @@ struct xocl_subdev_map {
 	{ XOCL_PCI_DEVID(0x10EE, 0x505D, PCI_ANY_ID, U55C_USER_RAPTOR2) },\
 	{ XOCL_PCI_DEVID(0x10EE, 0x5061, PCI_ANY_ID, U50LV_USER_RAPTOR2) },\
 	{ XOCL_PCI_DEVID(0x10EE, 0x506D, PCI_ANY_ID, U50C_USER_RAPTOR2) },\
+	{ XOCL_PCI_DEVID(0x10EE, 0x5075, PCI_ANY_ID, X3522PV_USER_RAPTOR2) },	\
```

Comment:
Please add a entry to  XOCL_DSA_DYNAMIC_MAP as well. Otherwise, xbmgmt will show empty name when shell package is not installed.